src/main/java/Main.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+
+public class Main {
+    public static final int MAX_TASK_CAPACITY = 100;
+    public static final String EXIT = "bye";
+    public static final String PRINT_TASK_LIST = "list";
+    public static final String TASK_DONE = "done";
+    public static final String ADD_EVENT = "event";
+    public static final String ADD_DEADLINE = "deadline";
+    public static final String ADD_TODO = "todo";
+
+
+    public static void main(String[] args) {
+
+        printGreetingMessage();
+
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[MAX_TASK_CAPACITY];
+        String command;
+        int numTask = 0;
+
+        while (true) {
+            command = in.nextLine();
+
+            if (command.equals(EXIT)) {
+                break;
+            } else if (command.equals(PRINT_TASK_LIST)) {
+                Task.printTaskList(tasks);
+                continue;
+            }
+            //the above are single word command
+
+            numTask = respondMultiWordCommand(tasks, command, numTask);
+        }
+
+
+        printExitMessage();
+    }
+
+    private static int respondMultiWordCommand(Task[] tasks, String command, int numTask) {
+        String[] words = command.split(" ");
+        if (words[0].equals(TASK_DONE)) {
+            tasks[Integer.parseInt(words[1]) - 1].markAsDone();
+        } else {
+            numTask = addTask(words[0], tasks, command, numTask);
+        }
+        return numTask;
+    }
+
+    private static void printExitMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void printGreetingMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    private static int addTask(String beginning, Task[] tasks, String command, int numTask) {
+        System.out.println("____________________________________________________________");
+        int dividerPosition;
+        System.out.println("Got it. I've added this task: ");
+        switch (beginning) {
+            case ADD_EVENT:
+                dividerPosition = command.indexOf("/at");
+                tasks[numTask] = new Event(command.substring(6, dividerPosition), command.substring(dividerPosition + 4));
+                break;
+            case ADD_DEADLINE:
+                dividerPosition = command.indexOf("/by");
+                tasks[numTask] = new Deadline(command.substring(9, dividerPosition), command.substring(dividerPosition + 4));
```

Comment:
This line is 122 characters long, should this be split into two lines instead?

Reply:
If you don't count the indentation spaces at the start of the line, it's 109 characters